pkg/gatherers/clusterconfig/dvo_metrics.go

```diff
@@ -0,0 +1,42 @@
+package clusterconfig
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+func (g *Gatherer) GatherDVOMetrics(ctx context.Context) ([]record.Record, []error) {
+	apiURL, err := url.Parse("http://deployment-validation-operator-metrics.deployment-validation-operator.svc:8383")
```

Comment:
I think it would be better to find the service name and port programmatically. In other words, rely only on the namespace name. 
Maybe we could also suggest renaming of the service - simply to `metrics`.

Reply:
@tremes Yeah, I suppose we shouldn't access it like this, but is there really any reason why/how it could change to anything else? Perhaps it would be worth bringing up on the next DVO sync. Dynamic service discovery seems to me like an additional complexity that may not really be necessary.

Reply:
Well my point was that you would never know when someone will change the name or the port in the [service](definition)- As for the name, it's true that we will have to rely either on the fact that there's only one service, or simply try to find a service whose name contains `metrics`.

Reply:
Okay, so I hopefully added that functionality in f201c50.